LICENSES/BSD-2-Clause.txt

```diff
@@ -0,0 +1,22 @@
+Copyright (c) <year> <owner>. All rights reserved.
```

Comment:
Is this intentionally left blank, or should it also say?
```suggestion
Copyright (c) 2020 Benedict Harcourt. All rights reserved.
```

Reply:
That's what the tool emits; even after asking for your name. I presume it's meant to be a template...? However, the [ones for the tool itself](https://github.com/fsfe/reuse-tool/tree/master/LICENSES) don't have attribution at all...